txpool/txpool.go

```diff
@@ -325,12 +324,16 @@ func (p *TxPool) SetSigner(s signer) {
 
 // SetSealing sets the sealing flag
 func (p *TxPool) SetSealing(sealing bool) {
-	p.sealing.Store(sealing)
+	if sealing {
+		atomic.StoreUint32(&p.sealing, 1)
```

Comment:
can't we stick with https://pkg.go.dev/sync/atomic#CompareAndSwapInt32 or https://pkg.go.dev/sync/atomic#Bool.CompareAndSwap with no `if` at all?

Reply:
Thank you for the advice! With CAS for bool, it's possible not to use if statement. But CompareAndSwapUint32 still needs unless changing the argument type to uint32. Am I correct?
For now, I fixed and please check whether I fixed as you expected: 6b162652e416c035c8038e04604552bd04cbee0f
Thank you

Reply:
we have updated ibft already.